### DISCUSSION TITLE
Temporal reference system - dataset (dataset series) interoperability metadata

### DISCUSSION THREAD

**Discussion by Ana-SGA** [INSPIRE-MIF/helpdesk#94] [Q&A] [answered]
Dear all & @heidivanparys , @sMorrone ,

can anyone confirm whether the element Temporal reference system as dataset (dataset series) **interoperability** metadata refers to element Temporal references (Temporal extent and Reference date (publication/creation/last revision)?

Thanks!
Ana

**Comment by fabiovinci** [comment, marked as answer]
Dear @Ana-SGA,

the answer to your question is "no" because the dates to be provided in the element _Temporal references_ (Temporal extent and Reference date (publication/creation/last revision) shall conform to the default temporal reference system, as stated in the IR 1205/2008, so there is no need to specify the temporal reference system.

![image](https://user-images.githubusercontent.com/22212406/165718973-a6411d94-b20e-4816-8ff1-a90f4b9bd225.png)

The element _Temporal reference system_ as dataset (dataset series) interoperability metadata is required by the IR 1089/2010, to document a different temporal reference system of the dates present in the dataset itself, only if other temporal reference systems are specified for a specific spatial data theme.

![image](https://user-images.githubusercontent.com/22212406/165721015-1683918c-84a7-43ea-8396-de46d2f30ad0.png)

For this reason, the metadata element _Temporal reference system_ is conditional.

![image](https://user-images.githubusercontent.com/22212406/165721521-7250896d-177b-472e-b62c-209c0fcc7691.png)

**Comment by Ana-SGA** [comment]
Dear @fabiovinci ,

> The element _Temporal reference system_ as dataset (dataset series) interoperability metadata is required by the IR 1089/2010, to document a different temporal reference system of the dates present in the dataset itself, only if other temporal reference systems are specified for a specific spatial data theme.
> 

thanks a lot. So, just to be sure, interoperability metadata element Temporal reference system refers only if in the data itself there are some dates (different from the default and according to the theme)? Can or does INSPIRE Validator recognize error if the dates in the data are in a different temporal reference system than stated in the metadata?

Kind regards,
Ana

**Comment by fabiovinci** [comment]
Dear @Ana-SGA,

yes to your first question, the interoperability metadata element `Temporal reference system` is mandatory only if the spatial data set or one of its feature types contains temporal information that does not refer to the Gregorian Calendar.

Regarding your second question, at the moment the INSPIRE Validator does not perform this type of check because the information is contained in different resources, i.e. when you validate a dataset the validator is not able to retrieve the information about the temporal reference system since it is contained in the metadata.

In any case, the INSPIRE Validator checks that dates are expressed in accordance with ISO 8601, as requested by Implementing rules.

**Comment by Ana-SGA** [comment]
Dear @fabiovinci ,

ok, thanks for all the information.

Kind regards,
Ana
